stl/inc/xstring

```diff
@@ -2200,10 +2200,11 @@ public:
     // length of internal buffer, [1, 16]:
     static constexpr size_type _BUF_SIZE = 16 / sizeof(value_type) < 1 ? 1 : 16 / sizeof(value_type);
     // roundup mask for allocated buffers, [0, 15]:
-    static constexpr size_type _ALLOC_MASK =
-        sizeof(value_type) <= 1
-            ? 15
-            : sizeof(value_type) <= 2 ? 7 : sizeof(value_type) <= 4 ? 3 : sizeof(value_type) <= 8 ? 1 : 0;
+    static constexpr size_type _ALLOC_MASK = sizeof(value_type) <= 1   ? 15
+                                             : sizeof(value_type) <= 2 ? 7

```

Comment:
Is there any chance to remove the additional 2 spaces before the `:`

Reply:
Not without disabling clang-format here.

Reply:
Resolving with no change since I don't see how to improve this.

Reply:
Actually, it looks like [clang-format 11](https://releases.llvm.org/11.0.0/tools/clang/docs/ClangFormatStyleOptions.html) has a new option to deal with this, `AlignOperands: AlignAfterOperator`